pkg/addons/default/aws_node_test.go

```diff
@@ -50,18 +48,18 @@ var _ = Describe("default addons - aws-node", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(awsNode.Spec.Template.Spec.Containers).To(HaveLen(1))
 			Expect(awsNode.Spec.Template.Spec.Containers[0].Image).To(
-				Equal("602401143452.dkr.ecr.eu-west-2.amazonaws.com/amazon-k8s-cni:v1.3.2"),
+				Equal("602401143452.dkr.ecr.eu-west-1.amazonaws.com/amazon-k8s-cni:v1.4.1"),
```

Comment:
Now that I think of it, new test doesn't actually trigger any changes, as `sample-1.12.json` comes with 1.4.1 already... let me see what can be done here.

Reply:
Yes, shouldn't we use 1.11 instead? (That's what I thought when I tried to make the tests relevant.)

Reply:
Ok, let's not worry about that, the test would just look kind of weird. I'll remove `sample-1.10.json`.